jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -202,10 +205,12 @@ YUI.add('entity-details', function() {
 
     render: function() {
       return (
-        <div className={this._generateClasses()}>
+        <div className={this._generateClasses()}
+          ref="content"
+          tabIndex="0">
```

Comment:
I don't believe so, all it's doing is setting the focus when the panel or entity details open. I don't think this can impact on the breadcrumbs at all, but I may have missed something. What do you think the issue might be?